### PR TITLE
Updated test_c8ql.py, test_cursor.py and changed customPrefix to custom_prefix

### PR DESCRIPTION
**Pull request by koshy-thomas**
## Description

Updated test_c8ql.py (also added sql test case), test_cursor.py and change customPrefix to custom_prefix

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**Test Configuration**:

* C8 Version:

## Reviews

Please identify two developers to review this change

- [x] @ricardo-macrometa 
- [x] @dlozina-macrometa 

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added new functional tests that prove my fix is effective or that my feature works
- [ ] Existing and new functional tests pass with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: c8/api.py [hunk 1]
```diff
@@ -79,7 +79,7 @@ def context(self):
         """
         return self._executor.context
 
-    def _execute(self, request, response_handler, customPrefix=None):
```
Comment (dlozina-macrometa) [inline c8/api.py:82]: Great stuff, this was poking my eyes ☺️ Pep8

**Review by dlozina-macrometa — APPROVED** [review]
Awesome, can't wait to run this part of the tests!

**Review by ricardo-macrometa — APPROVED** [review]

**Review by edgargarciamacrometa — COMMENTED** [review]
Awesome, test_c8ql works well, but when running test_cursor I receive the error:
E       fixture 'tst_fabric' not found
>       available fixtures: bad_col, bad_ecol, bad_fabric_name, bad_fvcol, bad_graph, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, client, col, docs, doctest_namespace, ecol, edocs, fvcol, fvdocs, geo, graph, lecol, monkeypatch, password, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, setup_collection, sys_fabric, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, tst_fabric_name, tvcol, tvdocs, username
>       use 'pytest --fixtures [testpath]' for help on them.

**Comment by koshy-thomas** [comment]
> Awesome, test_c8ql works well, but when running test_cursor I receive the error: E fixture 'tst_fabric' not found
> 
> > ```
> >   available fixtures: bad_col, bad_ecol, bad_fabric_name, bad_fvcol, bad_graph, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, client, col, docs, doctest_namespace, ecol, edocs, fvcol, fvdocs, geo, graph, lecol, monkeypatch, password, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, setup_collection, sys_fabric, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, tst_fabric_name, tvcol, tvdocs, username
> >   use 'pytest --fixtures [testpath]' for help on them.
> > ```

> Awesome, test_c8ql works well, but when running test_cursor I receive the error: E fixture 'tst_fabric' not found
> 
> > ```
> >   available fixtures: bad_col, bad_ecol, bad_fabric_name, bad_fvcol, bad_graph, cache, capfd, capfdbinary, caplog, capsys, capsysbinary, client, col, docs, doctest_namespace, ecol, edocs, fvcol, fvdocs, geo, graph, lecol, monkeypatch, password, pytestconfig, record_property, record_testsuite_property, record_xml_attribute, recwarn, setup_collection, sys_fabric, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory, tst_fabric_name, tvcol, tvdocs, username
> >   use 'pytest --fixtures [testpath]' for help on them.
> > ```

that is fixed in your PR -100 that's why I didn't add it in this PR (once your PR is merged you won't get error) it's about having another tst_fabric fixture which you added in your PR

**Review by jyuwei — APPROVED** [review]

**Pull request merged**